app/assets/stylesheets/application.css

```
@@ -13,3 +13,107 @@
  *= require_tree .
  *= require_self
  */
+
+ .checkin_header{
+    background-color: #2c2b37;
+    
+  }
+
+  .checkin_header nav{
+    padding-top: 10px;
+    padding-bottom: 10px;
+    margin-right: 50px;
+    margin-left: 50px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center; 
+    width: 90%;
```

Alphabetize your CSS

```suggestion
    align-items: center;
    display: flex;
    flex-direction: row;
    justify-content: space-between;
    margin-left: 50px;
    margin-right: 50px;
    padding-bottom: 10px;
    padding-top: 10px;
    width: 90%;
```